mappings/net/minecraft/client/gui/screen/VideoOptionsScreen.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS nj net/minecraft/client/gui/screen/VideoSettingsScreen
+CLASS nj net/minecraft/client/gui/screen/VideoOptionsScreen
```

Comment:
Change was made to match the current yarn name for it. They can be changed to XSettingsScreen instead if you want

Reply:
Do we want to change GameOptions & Options to GameSettings & Settings too then?

Reply:
VideoSettings makes more sense imo... remember this isn’t yarn

Reply:
If you want you can do GameOptions -> GameSettings for consistency, since it isn’t explicitly called that anywhere iirc

The GameOptions and Options are yours to decide the names of

Reply:
Changed. Options -> Settings can be decided on later.